src/util.rs

```diff
@@ -44,6 +44,25 @@ pub fn history() -> web_sys::History {
     window().history().expect("Can't find history")
 }
 
+/// Convenience function to access the `web_sys::HtmlDocument`.
+pub fn html_document() -> web_sys::HtmlDocument {
+    wasm_bindgen::JsValue::from(document()).unchecked_into::<web_sys::HtmlDocument>()
+}
+
+/// Convenience function to get all cookies from the current `HtmlDocument`
+#[cfg(feature = "cookies")]
+pub fn cookies() -> cookie::CookieJar {
+    let mut jar = cookie::CookieJar::new();
+    if let Ok(cookie_str) = html_document().cookie() {
+        for cs in cookie_str.split(";") {
+            if let Ok(c) = cookie::Cookie::parse_encoded(cs.to_string()) {
```

Comment:
Is `cs.to_string()` necessary? `parse_encoded` accepts `Into<Cow<'c, str>>` so maybe `cs.into()` is enough and we don't have to allocate memory for `String`.

Reply:
> Is `cs.to_string()` necessary?

yes, otherwise I get `borrowed value does not live long enough`